### DISCUSSION TITLE
How do i force my App to Run externally, No Shopify iframe

### DISCUSSION THREAD

**Discussion by emmanuelizzy** [osiset/laravel-shopify#807] [Ideas]
I have been searching and trying to figure out how to prevent my app from running in Shopify iframe. I have been able to disable the redirection by making `forceRedirect:false` but my app still runs in Shopify iframe. Please any pointers on how I can disable this?

Thanks

**Comment by arnedeutscher** [comment]
You have to define `SHOPIFY_APPBRIDGE_ENABLED=0` in your `.env` file.
Look at: [https://github.com/osiset/laravel-shopify/wiki/Installation#appbridge--legacy](https://github.com/osiset/laravel-shopify/wiki/Installation#appbridge--legacy)
